Week3/homework/step2-2.js

```diff
@@ -2,19 +2,34 @@
 
 function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
   const numbers = [];
-
   // Replace this comment and the next line with your code
   console.log(startIndex, stopIndex, threeCallback, fiveCallback, numbers);
+  for (let i = startIndex; i <= stopIndex; i++) {
+    numbers.push(i);
+  }
+  numbers.map(number => {
+    if (number % 3 === 0 && !(number % 5 === 0)) {
+      threeCallback(number);
+    } else if (number % 5 === 0 && !(number % 3 === 0)) {
+      fiveCallback(number);
+    } else if (number % 3 === 0 && number % 5 === 0) {
+      threeCallback(number);
+      fiveCallback(number);
+    }
+    return number;
+  });
 }
 
 function sayThree(number) {
   // Replace this comment and the next line with your code
   console.log(number);
+  return number + ' is divisible by three';
```

Comment:
There is not much point in returning this string concatenation, as there is no caller that uses this value. It would be far better to `console.log` this. Then, instead of this rather uninformative output:

```
10
12
15
15
```

we would get the very informative output:

```
10 is divisible by five
12 is divisible by three
15 is divisible by three
15 is divisible by five
```

```js
function sayThree(number) {
  console.log(number + ' is divisible by three');
}

function sayFive(number) {
  console.log(number + ' is divisible by five');
}
```